cypress/e2e/signIn.cy.js

```diff
@@ -2,6 +2,16 @@
 
 describe('Sign In page', () => {
   it('should provide an ability to log in', () => {
+    cy.visit('https://react-redux.realworld.io/#/login?_k=m07odl')
```

Comment:
Use relative URL instead of the full URL when visiting the 'Sign In' page. 
You can add base url to your config file. Start your test from that url point and then go to Sign page.